test/integration.js

```diff
@@ -257,33 +277,45 @@ tape('POST /channel/{id}/{validator-messages}: wrong signature', function(t) {
 	.catch(err => t.fail(err))
 })
 
-tape('POST /channel/{id}/{validator-messages}: wrong (deceptive) root hash', function(t) {
-	const stateRoot = '6def5a300acb6fcaa0dab3a41e9d6457b5147a641e641380f8cc4bf5308b16f1'
+tape('POST /channel/{id}/{validator-messages}: wrong (deceptive) root hash', function(t) {	
+	let deceptiveRootHash = ""
 
-	fetch(`${followerUrl}/channel/${dummyVals.channel.id}/validator-messages`, {
-		method: 'POST',
-		headers: {
-			'authorization': `Bearer ${dummyVals.auth.leader}`,
-			'content-type': 'application/json',
-		},
-		body: JSON.stringify({
-			"messages": [{ 
-				"type": 'NewState', 
-				stateRoot,
-				// the real tree is 11, 2
-				"balances": { "myAwesomePublisher" : "12", "anotherPublisher": "3" },
-				"lastEvAggr": "2019-01-23T09:10:29.959Z",
-				"signature": `Dummy adapter for ${stateRoot} by awesomeLeader`
-			}]
-		}),
+	fetch(`${followerUrl}/channel/${dummyVals.channel.id}/validator-messages/${dummyVals.ids.leader}/NewState?limit=1`)
+	.then(res => res.json())
+	.then(function(res) {
+
+		const { balances } = res.validatorMessages[0].msg
+		const fakeBalances = { "publisher": "3" }
+		// increase the state tree balance by 1
+		let incBalances = {}
```

Comment:
you don't need to increment; the same balances is also a valid state transition, cause it does not violate the OUTPACE rules